internal/usecase/util/access_token/access_token_test.go

```diff
@@ -0,0 +1,51 @@
+package access_token_test
+
+import (
+	"testing"
+	"time"
+	"todoapp/internal/entity"
+	"todoapp/internal/usecase/util/access_token"
+)
+
+func TestCreateToken(t *testing.T) {
+	user := entity.User{
+		Username: "daopham",
+	}
+
+	_, err := access_token.CreateToken(&user, time.Now())
```

Comment:
specify the specific time, and then check the token string

Suggestion:
createTime := time.Unix(...)

if token != "asdasd..." { t.Errorf(....) }